### PR TITLE
Add support for stored credentials for the checkout.com gateway

### PR DESCRIPTION
**Pull request by dbackowski**
This will add support for stored credentials for the checkout.com gateway

### CODE REVIEW

File: lib/active_merchant/billing/gateways/checkout_v2.rb [hunk 1]
```diff
@@ -77,6 +77,9 @@ def store(credit_card, options = {})
           r.process do
             post = {}
             post[:currency] = options[:currency]
+            post[:payment_type] = "Recurring" if credit_card.type == "visa"
```
Comment (himn1) [inline lib/active_merchant/billing/gateways/checkout_v2.rb:80]: hmm recurring payment only for VISA?

**Review by himn1 — COMMENTED** [review]
1 comment

File: lib/active_merchant/billing/gateways/checkout_v2.rb (hunk 1 above)
Comment (mizinsky) [reply lib/active_merchant/billing/gateways/checkout_v2.rb:80]: I can confirm that. From the Checkout docs:

> The initial cardholder-initiated transaction of a scheduled recurring series should not be marked as "Recurring" (unless it is a Visa card, in which case it must be marked as "Recurring") and it should have merchant_initiated set to false. All subsequent payments in the series should include payment_type: "Recurring", merchant_initiated: true and the previous_payment_id.

Comment (himn1) [reply lib/active_merchant/billing/gateways/checkout_v2.rb:80]: 🆒

**Review by himn1 — APPROVED** [review]

**Pull request merged**